packages/aragon-api/src/index.js

```diff
@@ -538,6 +556,76 @@ export class AppProxy {
         pluck('result')
       )
   }
+
+  /**
+   * send a forwarded action's information to the wrapper
+   *
+   * @param {string} actionId ID assigned to forwarded action in Forwarder
+   * @param {string} evmScript The execution script caught by the Forwarder
+   * @return {void}
+   */
+  newForwardedAction (actionId, evmScript) {
+    return this.rpc.send(
+      'update_forwarded_action',
+      [actionId, evmScript]
+    )
+  }
+
+  /**
+   * update a forwarded action's state the wrapper
+   *
+   * If the execution is pending, state = 0
+   * If the execution completed, state = 1
+   * If the execution failed, state = 2
+   *
+   * @param {string} actionId ID assigned to forwarded action in Forwarder
+   * @param {integer} state The current state of the forwarded action within the forwarder
```

Comment:
I'm not super happy with this interface; we should probably use string constants instead (can't use symbols because they don't go over the RPC wire).